### PR TITLE
Update controller to 8.1.133

### PR DESCRIPTION
**Pull request by cwmoriarty**
# Proposed Changes

> Update controller to 8.1.133 (current). Tested locally.

### CODE REVIEW

**Comment by cwmoriarty** [comment]
FWIW I have not yet tested this with the changes in `cfergeau:base9`.

**Comment by cfergeau** [comment]
> FWIW I have not yet tested this with the changes in `cfergeau:base9`.

In my opinion it's best to get this in first and make a release as this PR is trivial compared to mine :)

**Comment by renewoensdregt** [comment]
@cwmoriarty You beat me to it, but this build failed because 17.98 E: Version '2.34-6ubuntu1.8' for 'binutils' was not found

You need to change it to 2.34-6ubuntu1.9

**Comment by frenck** [comment]
> You need to change it to 2.34-6ubuntu1.9

Not really in this PR though....

**Comment by cwmoriarty** [comment]
> @cwmoriarty You beat me to it, but this build failed because 17.98 E: Version '2.34-6ubuntu1.8' for 'binutils' was not found
> 
> You need to change it to 2.34-6ubuntu1.9

Definitely support merging your PR first.

**Comment by cwmoriarty** [comment]
> Maybe I am not clear on full git processes yet. Would you not include all changes in one PR to make the new version work? If this PR is merged as-is, it will not work. I can submit a fully working PR soon I think with both changes, but if that is not the right process, let me know. Learning as we go.

I think best practice here is to sync your branch with mine and resolve the changes. I have added the correct binutils pin.

**Comment by frenck** [comment]
> Would you not include all changes in one PR to make the new version work?

It isn't related.

**Comment by renewoensdregt** [comment]
thank you! Makes sense. Thought of mentioning it as that is quicker for you to change vs me updating and requesting a merge. Anyway I see it built correctly now after the update so we should be fine.
________________________________

**Comment by cfergeau** [comment]
> > Would you not include all changes in one PR to make the new version work?
> 
> It isn't related.

It's definitely not related, but since it's separated in its own commit, and since there aren't multiple branches in flux which would need this change, then it's in my opinion acceptable and less cumbersome for the contributor to have this fix together with a PR doing something different.

File: unifi/Dockerfile [hunk 1]
```diff
@@ -10,14 +10,14 @@ RUN \
     apt-get update \
     && apt-get upgrade -y \
     && apt-get install -y --no-install-recommends \
-        binutils=2.34-6ubuntu1.8 \
+        binutils=2.34-6ubuntu1.9 \
```
Comment (frenck) [inline unifi/Dockerfile:13]: Please remove this from this PR.

**Comment by renewoensdregt** [comment]
> Sorry, I don't agree. So, there is that.
Time to use my favorite quote at work.... "You´re not paid to like it "

**Comment by cwmoriarty** [comment]
@frenck What's next here?

**Comment by renewoensdregt** [comment]
As I understand a separate PR to pin binutils 1.9 as he reverted the change and asked for a separate one.
________________________________

**Comment by renewoensdregt** [comment]
Created a separate pull request 496. Hopefully this will help and we can have the new version soon.
________________________________

**Comment by frenck** [comment]
@renewoensdregt You really need to have more patience...

**Comment by frenck** [comment]
> @frenck What's next here?

Wait? There is no fire to get this out in a rush?

**Comment by renewoensdregt** [comment]
@frenck There is no fire, but please understand that not everyone is as clear on Git and the processes are you are. I´m learning and really want to help, but if there is no explanation, clarification on what to do vs just a comment that you disagree, that does not really help me to do what you expect needs to be done. Little bit of empathy would be appreciated :-)

**Comment by frenck** [comment]
> Little bit of empathy would be appreciated :-)

This is not a school.

**Review by frenck — APPROVED** [review]
Thanks, @cwmoriarty 👍 

../Frenck

**Comment by cfergeau** [comment]
> > Little bit of empathy would be appreciated :-)
> 
> This is not a school. There are tons of resources out there to learn this stuff.

@renewoensdregt is not clear on the processes expected to be used in addon-unifi, and hassio-addons in general, which are - as far as I understand - based on how you work. I'm not sure this is explicitly documented? I had the same confusion regarding your expectations for the binutils bump.

In general, when people understand the processes, they can file PRs in the right way from the start, resulting in 
- less work for you because you did not need to do the changes/testing/... yourself
- less PR review work/less discussions, because people were aware of how things should be when filing the PR
- ideally, they can even help review other PRs so that they get  in the shape you expect

**Comment by frenck** [comment]
> I'm not sure this is explicitly documented? 

Keep PR small and to the point. That is a general rule in software development and doing PR anywhere. You should not need documentation for that.

> n general, when people understand the processes, they can file PRs in the right way from the start, resulting in

You are making this way bigger and complex than it should be. Also, again, noise. 

STOP IT.

Thanks 👍

**Comment by buzo-ffm** [comment]
Minor detail: The version number is 8.1.113, not 133, and [these are the release notes](https://community.ui.com/releases/UniFi-Network-Application-8-1-113/af46fd38-8afe-4cef-8de1-89636b02b52c).

**Pull request merged**